From 05cc932957034e2295aec1b96388db6a9a252f0e Mon Sep 17 00:00:00 2001
From: xcore995 <arman.p95@gmail.com>
Date: Tue, 21 Apr 2020 06:50:21 -0400
Subject: [PATCH] Fix Tethering

---
 server/NatController.cpp | 14 +++++++-------
 1 file changed, 7 insertions(+), 7 deletions(-)

diff --git a/server/NatController.cpp b/server/NatController.cpp
index 58c732d0..8876cc5b 100644
--- a/server/NatController.cpp
+++ b/server/NatController.cpp
@@ -201,13 +201,13 @@ std::string NatController::makeTetherCountingRule(const char *if1, const char *i
 int NatController::setForwardRules(bool add, const char *intIface, const char *extIface) {
     const char *op = add ? "-A" : "-D";
 
-    std::string rpfilterCmd = StringPrintf(
-        "*raw\n"
-        "%s %s -i %s -m rpfilter --invert ! -s fe80::/64 -j DROP\n"
-        "COMMIT\n", op, LOCAL_RAW_PREROUTING, intIface);
-    if (iptablesRestoreFunction(V6, rpfilterCmd) == -1 && add) {
-        return -1;
-    }
+    //std::string rpfilterCmd = StringPrintf(
+     //   "*raw\n"
+     //   "%s %s -i %s -m rpfilter --invert ! -s fe80::/64 -j DROP\n"
+     //   "COMMIT\n", op, LOCAL_RAW_PREROUTING, intIface);
+    //if (iptablesRestoreFunction(V6, rpfilterCmd) == -1 && add) {
+    //    return -1;
+   // }
 
     std::vector<std::string> v4 = {
         "*filter",
-- 
2.17.1

